src/Admin/Pool.php

```diff
@@ -85,6 +85,11 @@ class Pool
      */
     private $templateRegistry;
 
+    /**
+     * @var AdminInterface
+     */
+    private $currentAdmin;
```

Comment:
:-1: The pool should be kept stateless. IMHO the original idea for this class was some kind of registry for all admins.

Reply:
I can move it to utils. IMO all things based on `Request` (like `$request->get('uniqid')`, `csrf` ...) should be do it on it.